Celeste.Mod.mm/Mod/UI/OuiMapSearch.cs

```diff
@@ -179,15 +179,7 @@ private class SearchMenu : Entity {
 
             } else if (c == (char) 127) {
```

Comment:
This branch of the if should just be removed outright IMO, there's no reason to keep it.

Reply:
The reason I left it is because I saw several other files leave the branch alone:
https://github.com/EverestAPI/Everest/blob/611172ef3cc4b2d486db69d57e77fc9640e5c08a/Celeste.Mod.mm/Patches/OuiFileNaming.cs#L45-L48
Going to remove it now.